pylint/checkers/base/docstring_checker.py

```diff
@@ -73,6 +73,11 @@ class DocStringChecker(_BasicChecker):
             "docstring.",
             {"old_names": [("C0111", "missing-docstring")]},
         ),
+        "C0150": (
+            "%s",
```

Comment:
Could we make this message a little more descriptive? Something like:

```suggestion
            "Disallowed word in docstring: '%s'",
```

I know the original `fixme` message doesn't have this but I don't think that's nice :smile: